typescript/web/src/components/auth-manager/signin-modal/signin-modal.tsx

```diff
@@ -140,6 +143,9 @@ export const SigninModal = ({
 
               <DividerWithText>or sign in with email</DividerWithText>
               <form
+                css={{
+                  ":invalid": { button: { color: "red" } },
```

Comment:
Is this used for something?

Reply:
Oh no, good catch. This was from a -failed- attempt to use native CSS selector to manage the `:valid`/`:invalid` state of the form